pkg/lockutil/lockutil_windows.go

```diff
@@ -38,8 +38,8 @@ var (
 
 const (
 	// see https://msdn.microsoft.com/en-us/library/windows/desktop/aa365203(v=vs.85).aspx
-	LOCKFILE_EXCLUSIVE_LOCK   = 0x00000002
-	LOCKFILE_FAIL_IMMEDIATELY = 0x00000001
+	LOCKFILE_EXCLUSIVE_LOCK   = 0x00000002 //nolint:revive,staticcheck // var-naming: it's ok to use ALL_CAPS here
+	LOCKFILE_FAIL_IMMEDIATELY = 0x00000001 //nolint:revive,staticcheck // var-naming: it's ok to use ALL_CAPS here
```

Comment:
The comment should explain why this is ok here. I guess because we want to keep the original names so it is easy to search the Windows documentation?

Reply:
Removed `//nolint` comment.